File: C/21.cpp
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main(void){
	setlocale(LC_ALL, "Portuguese");
	float soma,item,i,n;
	printf("Esse programa determina a soma entre N itens!\n\n");
	
	printf("\n\nQuantidade de itens: ");
	scanf("%f",&n);
	i=1;
	do{
		printf("\n\nPreço do item %.0f: ",i);
		scanf("%f",&item);
		soma=soma+item;
		i++;
	}while(i<=n);
	printf("\n\nA soma do(s) %.0f itens é de %.2f",n,soma);
 
	printf("\n\nObrigado por usar nosso sistema!\n\n");
	system("pause");
	return 0;
}

File: Python/21.py
printf = print
printf("Esse programa determina a soma entre N itens!\n\n")

n = int(input("\n\nQuantidade de itens: "))
i = 1
soma = 0
while (i <= n):
    item = float(input(f"\n\nPreço do item {i}: "))
    soma = soma + item
    i = i + 1

printf(f"\n\nA soma do(s) {n} itens é de {soma}")

printf("\n\nObrigado por usar nosso sistema!\n\n")
